docs/detections/rules-ui-create.asciidoc

```diff
@@ -255,6 +266,21 @@ field values.
 +
 . Continue with <<preview-rules, previewing the rule>> (optional) or click *Continue* to <<rule-ui-basic-params, configure basic rule settings>>.
 
+[discrete]
+[[create-new-terms-rule]]
+==== Create a new terms rule
```

Comment:
```suggestion
==== Create a New Terms rule
```
proper noun?

Reply:
My vote would be lowercase here, following the general usage of the other rule type names.